CHANGELOG.md

```diff
@@ -10,6 +10,7 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 - Bump `org.apache.commons:commons-lang3` from 3.14.0 to 3.15.0 ([#14861](https://github.com/opensearch-project/OpenSearch/pull/14861))
 
 ### Changed
+- Optimize _cat/nodes api ([#14853](https://github.com/opensearch-project/OpenSearch/pull/14853))
```

Comment:
Can we please make the changelog more descriptive? Something like 'Optimize _cat/nodes api by ....'